packages/hdwallet-provider/src/constructor/getOptions.ts

```diff
@@ -140,8 +134,10 @@ export const getOptions = (
   } else if (matchesLegacyArguments(args)) {
     return fromArguments(args);
   } else {
-    throw new Error(
-      "Unknown arguments format passed to new HDWalletProvider. Please check your configuration and try again"
-    );
+    const message =
+      "Unknown arguments format passed to new HDWalletProvider." +
+      " Please ensure you passed both provider information and a menmonic or " +
```

Comment:
This message is a little ambiguous.  Potential rewording of the second sentence to be less ambiguous (asusming I've got the right reading!): "Please ensure you passed provider information along with either a mnemonic or private keys."  OTOH if you want to leave it alone it's probably fine.

Reply:
Yeah I'll reword this